config/channel/post-install/storage-version-migrator.yaml

```diff
@@ -35,6 +35,6 @@ spec:
       restartPolicy: OnFailure
       containers:
         - name: migrate
-          image: ko://knative.dev/pkg/apiextensions/storageversion/cmd/migrate
```

Comment:
no - let's not do that.
reference here: https://github.com/knative/serving/pull/11229